lib/pry-doc.rb

```diff
@@ -23,4 +18,8 @@ def self.root
   root
 end
 
+require 'yard'
+require 'pry-doc/version'
+require 'pry-doc/pry_ext/method_info'
```

Comment:
How is this change relevant to this PR and what's the motivation?

Reply:
@kyrylo i explain in the commit message https://github.com/pry/pry-doc/pull/85/commits/968b073ce9ea077e8640ebf613c176422c11f96c

```
require files after PryDoc class is configured

This is because we need to ensure PryDoc.root is defined before loading other classes that depend on it
```

Reply:
@kyrylo tests started to fail as i was setting up `RubySourceInstaller` in the class body, and `RubySourceInstaller` needed access to the `PryDoc.root` method - and all this happened BEFORE `PryDoc.root` was defined when the requires were above the definition, so i just moved the requires under